### PR TITLE
Kokoro build configuration

### PR DESCRIPTION
**Pull request by stevemessick**
First cut at defining build files for Kokoro. This version does nothing. It will be useful to verify the config is correct before digging into the actual build mechanics.

@athomas @devoncarew @keertip

### CODE REVIEW

File: kokoro/macos_external/continuous.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+
+# Location of the bash script. Should have value <github_scm.name>/<path_from_repository_root>.
+# github_scm.name is specified in the job configuration (next section).
+build_file: "flutter-intellij-kokoro/kokoro/macos_external/kokoro_build.sh"
```
Comment (devoncarew) [inline kokoro/macos_external/continuous.cfg]: `flutter-intellij/kokoro/macos_external/kokoro_build.sh` ?

File: kokoro/macos_external/kokoro_build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Display commands being run.
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+
```
Comment (devoncarew) [inline kokoro/macos_external/kokoro_build.sh]: nit: extra eol

File: kokoro/macos_external/kokoro_build.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd ${KOKORO_ARTIFACTS_DIR}/github/flutter-intellij-kokoro
+./tool/kokoro/build.sh
```
Comment (devoncarew) [inline kokoro/macos_external/kokoro_build.sh]: nit add a preceding eol?

File: kokoro/macos_external/kokoro_build.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd ${KOKORO_ARTIFACTS_DIR}/github/flutter-intellij-kokoro
```
Comment (devoncarew) [inline kokoro/macos_external/kokoro_build.sh]: Just clarifying - from reading the above, the job name is `flutter-intellij-kokoro`?

File: kokoro/macos_external/continuous.cfg (hunk 1 above)
Comment (devoncarew) [reply kokoro/macos_external/continuous.cfg]: From reading the comment, I don't see `github_scm.name` specified?

> github_scm.name is specified in the job configuration (next section)

File: kokoro/macos_external/presubmit.cfg [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+
```
Comment (devoncarew) [inline kokoro/macos_external/presubmit.cfg]: bit: extra eol

File: kokoro/macos_external/continuous.cfg (hunk 1 above)
Comment (stevemessick) [reply kokoro/macos_external/continuous.cfg]: That quote is from the sample file generated by the code lab. The next section of the code lab describes the job configuration. I should clean that up.

File: kokoro/macos_external/kokoro_build.sh (hunk 2 above)
Comment (stevemessick) [reply kokoro/macos_external/kokoro_build.sh]: I just copied the file from the code lab, but I can remove that.

File: kokoro/macos_external/kokoro_build.sh (hunk 4 above)
Comment (stevemessick) [reply kokoro/macos_external/kokoro_build.sh]: Yes. It is the directory into which this repo is checked out when Kokoro runs.

**Pull request merged**